src/IMDb.Infrastructure/Repositories/AbstractRepository.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using IMDb.Core.Models;
+using IMDb.Core.Repositories;
+using Marten;
+
+namespace IMDb.Infrastructure.Repositories
+{
+    public abstract class AbstractRepository<T> : IRepository<T>
+    {
+        public string Url
```

Comment:
Make this `public string Url => GetUrl();` - it's the same thing, just a bit of syntactical sugar